controllers/ovncontroller_controller.go

```diff
@@ -302,3 +264,11 @@ func newDaemonset(cr *neutronv1.OVNController, cmName string, templatesConfigHas
 
 	return &daemonSet
 }
+
+func (r *OVNControllerReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	return ctrl.NewControllerManagedBy(mgr).
+		For(&neutronv1beta1.OVNController{}).
+		Owns(&corev1.Service{}).
+		Owns(&corev1.ConfigMap{}).
```

Comment:
Won't we want to watch the Daemonset too here? I don't see that in the existing code but perhaps adding it is good?

Reply:
Done thanks